README.md

```diff
@@ -1,3 +1,36 @@
+# Contribution Guide
+
+## Commit Code
+
+**deutsch**
+
+- Fork erstellen
+- Änderungen auf den fork commiten
+- Merge request machen
+- Pull Request reviewen lassen
+
+
+**english**
+
+- create a fork
+- commit changes to the fork
+- create a merge request
+- let the pull request be reviewed
+
+## Bug Report
+
+**deutsch**
+
+Falls du eine Issue anlegst sollte dieses einen klaren Titel mit einer genauen Beschreibung des enthalten. Du solltest so viele relevante Informationen wie möglich angeben und auch ein Code Beispiel, welches den Fehler demonstriert. Ein Bug Report soll dir und anderen helfen  den Fehler zu reproduzieren und einen Fix zu entwickeln.
+
+Bitte denk daran, dass ein Bug Report erstellt werden in der Hoffnung andere mit dem selben Problem zusammenzubringen und dann zusammen eine Lösung zu finden. Erwarte nicht, dass der Bug Report automatisch als Aufgabe gesehen wird oder sofort gelöst wird. Das Erstellen eines Bug Reports soll dabei helfen dir und anderen den ursprung des Problems zu ermitteln.
```

Comment:
> Bitte denk daran, dass ein Bug Report erstellt werden in der Hoffnung andere mit dem selben Problem zusammenzubringen und dann zusammen eine Lösung zu finden.

Bitte denk daran, dass Bug Reports erstellt werden, in der Hoffnung andere mit dem selben Problem zusammenzubringen und dann zusammen eine Lösung zu finden.

> anderen den ursprung

"den Ursprung"